c7n/filters/core.py

```diff
@@ -25,6 +25,7 @@
 from dateutil.tz import tzutc
 from dateutil.parser import parse
 import jmespath
+import ipaddress
```

Comment:
i'd like to localize the import to utils, also mu.custodian_archive needs updating with a new dependency, for lambda usage.